File: 6/stringreview.py
#!python

n = int(input().strip())


def walk(line):
    first = []
    second = []
    for i, c in enumerate(line):
        if i % 2 == 0:
            first.append(c)
        else:
            second.append(c)
    out = "".join(first) + " " + "".join(second)
    return out


for i in range(n):
    line = input()
    print(walk(line))

File: 6/stringreview.clj
(require 'clojure.string)

(let [N_t (read-line) 
      N (Integer/parseInt N_t)]
  (doseq [i (range N)]
    (let [line (read-line) 
          first (take-nth 2 line)
          second (take-nth 2 (rest line))]
      (println (clojure.string/join
                [(clojure.string/join first) 
                 \space
                 (clojure.string/join second)])))))


